pkg/scheduler/plugins/util/util.go

```diff
@@ -298,3 +298,48 @@ func NormalizeScore(maxPriority int64, reverse bool, scores []api.ScoredNode) {
 		scores[idx].Score = score
 	}
 }
+
+// GetAllocatedResource returns allocated resource for given job
```

Comment:
The jobinfo has `Allocated` field, whether it is ok in this case.

Reply:
Yes, but it is initial status when used in `OnSessionOpen`, namely, it's nil.

Reply:
got it.